deal.II-toolchain/packages/petsc.package

```diff
@@ -35,6 +35,9 @@ CONFOPTS="
   --with-mpi=1
   --with-x=0
   --with-64-bit-indices=0
+  COPTFLAGS='-O3 -march=native -mtune=native'
+  CXXOPTFLAGS='-O3 -march=native -mtune=native'
+  FOPTFLAGS='-O3 -march=native -mtune=native'
```

Comment:
Using `native` is always dangerous if you're working on clusters where the front-end nodes may have different processors than the compute nodes. Or if you have a collection of machines that you work on that are not all the same.

I don't know if that's the typical case for candi, but it's a point worth discussing.